### PR TITLE
added mouse behavior assays

### PR DESCRIPTION
**Pull request by amapeters**
elevated plus maze test, open field test, rotarod performance test, contextual fear conditioning test

### CODE REVIEW

File: synapseAnnotations/data/experimentalData.json [hunk 1]
```diff
@@ -12,6 +12,26 @@
...
+        "desription": "A test utilizing an enclosure consisting of a single area bordered by clear or opaque walls, used to measure movement, exploratory behavior, etc of an experimental subject."
+        "source": "http://purl.obolibrary.org/obo/MMO_0000258"
+      },
+      {
+        "value": "rotarod performance test"
+        "description": "A method that utilizes a device consisting of a rotating rod the speed of which is mechanically driven and precisely controlled, for instance held constant or accelerated. The rotarod test is often used to measure riding time or endurance in order to evaluate balance, motor coordination or grip strength."
+        "source": "http://purl.obolibrary.org/obo/MMO_0000567"
+      },
+      {
+        "value": "contextual fear conditioning test"
```
Comment (karawoo) [inline synapseAnnotations/data/experimentalData.json]: Just to double-check, this is different than our existing "contextual conditioning behavior"?
https://github.com/Sage-Bionetworks/synapseAnnotations/blob/32dc52fa25970ac82a3aa46fb25c2d20eb944170/synapseAnnotations/data/experimentalData.json#L250-L252

**Comment by amapeters** [comment]
@karawoo - please review

File: synapseAnnotations/data/experimentalData.json (hunk 1 above)
Comment (karawoo) [reply synapseAnnotations/data/experimentalData.json]: @amapeters could you follow up on this? We'd like to get all outstanding issues merged so we can do a new release of the annotations before the data release.

Comment (karawoo) [reply synapseAnnotations/data/experimentalData.json]: @amapeters do we need this for the November data release?

**Pull request merged**